### ISSUE TITLE
/v1/notify-deregistrations ip-address/ipv-4-address property requires a space in the OAS

### ISSUE THREAD

**Issue by PrathibaJee** [openBackhaul/RegistryOffice#255]
In the latest https://raw.githubusercontent.com/openBackhaul/RegistryOffice/develop/RegistryOffice.yaml , we have the following errors while loading the OAS specification in the [swagger editor](https://editor.swagger.io/) to generate server side stub , 

- Structural error at paths./v1/notify-deregistrations.post.requestBody.content.application/json.schema.properties.subscriber-address.properties.ip-address
should NOT have additional properties
additionalProperty: ipv-4-address
Jump to line 2387

- Structural error at paths./v1/notify-deregistrations.post.requestBody.content.application/json.schema.properties.subscriber-address.properties.ip-address.properties
should be object
Jump to line 2391

**proposal :**
![image](https://user-images.githubusercontent.com/62344562/204103204-3ecee499-c153-4ba2-841d-ec43f041b7fa.png)

**Comment by openBackhaul** [comment]
Cannot find any occurrences of wrong indentation of _ipv-4-address:_ anymore.
Swagger editor does not indicate a problem in 2387 or 2391.
Maybe has been fixed during YAML linting.
@PrathibaJee : If the problem would still occur during loading the OAS specification in the swagger editor, please assign back to me.

**Comment by PrathibaJee** [comment]
Sorry for creating an issue. Looks like I checked this in an outdated OAS version. I cannot able to see the problem now. Hence closing this issue. Thanks.

This was resolved before 3 days itself in the pull request [Minimizing the ActionProfile](https://github.com/openBackhaul/RegistryOffice/commit/383444e427b41365c3ff4bbcec38d011d580b6fb#diff-777d079469e77d51bf3d783452f18814c0788a353dbc135619e8da0186df2071)
![image](https://user-images.githubusercontent.com/62344562/204269558-bbf0dcf8-c8c7-4eec-9083-1f34c0dfc5ca.png)
